src/mixins/date-mixin.ts

```diff
@@ -65,7 +65,7 @@ export default class DateMixin extends Vue {
   }
 
   /**
-   * Converts a Date object to a date string (YYYY-MM-DD) in Pacific timezone.
```

Comment:
This is key information. Please restore it.

(The resulting yyyy-mm-dd may be different in a different timezone.)